compiler/quilt/tools/store.py

```diff
@@ -96,6 +109,9 @@ def __init__(self, location=None):
             )
             raise StoreException(msg.format(self._path))
 
+    def __eq__(self, rhs):
```

Comment:
If you define it, you also need to define `__ne__` and `__hash__`.

(`__ne__` can be just `not (self == rhs)`, and `__hash__` can be `hash(self._path)`)